components/support/rc_crypto/nss/systest/build.rs

```diff
@@ -58,6 +58,7 @@ fn main() {
     cfg.skip_struct(|s| {
         s == "SECKEYPublicKeyStr_u" // inline union
     });
+    cfg.skip_fn(|s| s == "PK11_CreateContextBySymKey");
```

Comment:
If I comment out this line, the tests still seem to pass for me locally - what errors were you seeing that led to this being skipped?

Reply:
Heh - want I actually want to do is try *breaking* something in the `nss_sys` crate and understand how this test helps me detect that, so I was poking around in here to see what I needed to do to start.